src/components/AirdropWalletsList.tsx

```diff
@@ -31,12 +33,14 @@ export const AirdropWalletsList: FC<Props> = props => {
   return (
     <View style={styles.container}>
       <Text style={styles.header}>{title}</Text>
-      <FlatList
-        data={wallets}
-        renderItem={({ item }) => (
-          <AirdropWalletsListItem wallet={item} callToAction={itemCallToAction && itemCallToAction(item)} />
-        )}
-      />
+      {wallets.length > 0 &&
+        wallets.map((wallet, index) => (
+          <AirdropWalletsListItem
+            key={index}
```

Comment:
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318 use wallet.id as key, I believe that there is a tslint rule for that you can add for this so it would display warning

Reply:
I think it s an antipattern only when you can remove elements, but will take a look, sure :)

Reply:
I think there is no point risking if we already have a unique id that we can use

Reply:
true, done